src/main/java/org/eclipse/opensmartclide/dbapi/controller/DeploymentController.java

```diff
@@ -64,15 +76,22 @@ public ResponseEntity<Deployment> updateDeployment(@PathVariable("id") String id
             if (deploymentData.isPresent()) {
                 Deployment _deployment = deploymentData.get();
                 _deployment.setUser_id(deployment.getUser_id());
+                _deployment.setUser(deployment.getUser());
```

Comment:
It is better to not use underscore ('_') before variable names, as suggested [here](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html).

Reply:
The name has been changed in commit 939785d98cb14c3882312537306fc9631a27260b .